drydock_provisioner/orchestrator/__init__.py

```diff
@@ -468,13 +468,37 @@ def execute_task(self, task_id):
 
 
             if len(node_networking_task.result_detail['successful_nodes']) > 0:
-                self.logger.info("Found %s successfully networked nodes, starting deployment." %
-                                 (len(node_networking_task.result_detail['successful_nodes'])))
+                self.logger.info("Found %s successfully networked nodes, configuring platform." %
+                                (len(node_networking_task.result_detail['successful_nodes'])))
+
+                node_platform_task = self.create_task(tasks.DriverTask,
+                                            parent_task_id=task.get_id(), design_id=design_id,
+                                            action=hd_fields.OrchestratorAction.ApplyNodePlatform,
+                                            task_scope={'site': task_site,
+                                                        'node_names': node_networking_task.result_detail['successful_nodes']})
+                self.logger.info("Starting node driver task %s to configure node platform." % (node_platform_task.get_id()))
+
+                node_driver.execute_task(node_platform_task.get_id())
+
+                node_platform_task = self.state_manager.get_task(node_platform_task.get_id())
+
+                if node_platform_task.get_result() in [hd_fields.ActionResult.Success,
```

Comment:
Both conditions if and elif check whether the result includes `hd_fields.ActionResult.PartialSuccess`. Meaning that:

```python

elif node_platform_task.get_result() in [hd_fields.ActionResult.Failure,
                                                              hd_fields.ActionResult.PartialSuccess]:
```

Will never be true for `hd_fields.ActionResult.PartialSuccess` as it was already  true for the condition before it.